aws-samples/amazon-chime-sdk-classroom-demo
# Fix classroom demo depedencies

**Issue #: N/A**

**Description of changes:**
Since TestCafe and Electron are not compatible, we decided to remove the TestCafe related script & dependencies. Now the demo should work as expected.

**Testing**

1. How did you test these changes?
* Run the app
* Select teacher on home screen and create classroom
* Run another instance of the app, then select student and then join the same classroom
* Start video and see if it appears on the remote side and vice versa
* Mute & unmute both of the student and teacher to see if it works
* Meanwhile also check for any possible console errors

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
